sys/dev/pci/pci_user.c

```diff
@@ -121,7 +123,14 @@ struct pci_conf_io64 {
 	uint32_t		status;		/* request status */
 };
 
+struct pci_list_vpd_io64 {
```

Comment:
Hmm, I guess it's debatable how much we really need the 64-bit one if we think that tools like pciconf are likely to be purecap rather than freebsd64.  (This series is making that decision of purecap only for NFS servers for example.)  I think using 32-bit system utils under 64-bit is probably less likely than using hybrid ones under cheri though in general.  The only thing I know of that uses this ioctl is `pciconf -lV`.  Most of the 32-bit compat shims for PCI were motivated by Y!BSD and 4.x pciconf binaries didn't invoke this ioctl.